src/controller/artifact/processor/processor.go

```diff
@@ -66,7 +68,9 @@ func Get(mediaType string) Processor {
 	// no registered processor found, use the default one
 	if processor == nil {
 		log.Debugf("the processor for media type %s not found, use the default one", mediaType)
-		processor = &defaultProcessor{mediaType: mediaType}
+		processor = &defaultProcessor{
```

Comment:
As we remove the `mediaType` property from the default processor, the default processor can be a singleton and we don't need to create it every time

Reply:
We have put default processor in processor registry. And we will get processor from it.So is it already singleton mode? I think it is.Do I need to modify code here? @ywk253100

Reply:
I have modified this part, PTAL again. Thanks!